### ISSUE TITLE
Destroying a girder with bullets crashes the server.

### ISSUE THREAD

**Issue by Sauberstaub** [space-wizards/space-station-14#234]
**Steps to reproduce:**
1. Shoot a wall with a projectile weapon until the girder is exposed.
2. Shoot the girder until it is destroyed.

**Comment by Injazz** [comment]
After few tests I came to conclusion it's because of weird interaction between wreckage spawn code and physics system. If wreckage has Physics component, it gonna crash on spawn. \
https://github.com/space-wizards/space-station-14/blob/master/Content.Server/GameObjects/Components/Damage/DestructibleComponent.cs#L57
causes server to crash

trying 
```
if (Owner.EntityManager.TrySpawnEntityAt(spawnOnDestroy, Owner.Transform.GridPosition, out var wreck)
```
returns false

trying
```
Owner.EntityManager.ForceSpawnEntityAt(spawnOnDestroy, Owner.Transform.GridPosition);
```
crashes the server
calling `var wreck = Owner.EntityManager.SpawnEntity(spawnOnDestroy);` without changing wreckage position works, but wreckage obviously spawns at origin point

**Comment by PJB3005** [comment]
Fixed by space-wizards/RobustToolbox/pull/821
